templates/config/production.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "${instanceName}",
+  "services": {
+    "repository": "https://github.com/OpenTermsArchive/${instanceName}-declarations.git",
+    "declarationsPath": "./declarations"
+  },
+  "recorder": {
+    "versions": {
+      "storage": {
+        "git": {
+          "snapshotIdentiferTemplate": "https://github.com/OpenTermsArchive/${instanceName}-snapshots/commit/%SNAPSHOT_ID",
+          "repository": "git@github.com:OpenTermsArchive/${instanceName}-versions.git",
+          "path": "./data/versions"
+        }
+      }
+    },
+    "snapshots": {
+      "storage": {
+        "git": {
+          "repository": "git@github.com:OpenTermsArchive/${instanceName}-snapshots.git",
+          "path": "./data/snapshots"
+        }
+      }
+    }
+  },
+  "fetcher": {
+    "language": "en"
+  },
+  "notifier": {
+    "sendInBlue": {
+      "updatesListId": ${sendinblueUpdateListId},
```

Comment:
Quotes missing

Reply:
updateListsId is supposed to be a number, as expected by [the SIB API](https://developers.sendinblue.com/reference/createcontact)